image.go

```diff
@@ -720,187 +1027,19 @@ func (i *containerImageRef) NewImageSource(_ context.Context, _ *types.SystemCon
 		}
 		// Add a note in the manifest about the layer.  The blobs are identified by their possibly-
 		// compressed blob digests.
```

Comment:
This looks like a comment that `addLayer()` should have in it, or as part of its godoc.